### PR TITLE
Review code from previous project

### PR DESCRIPTION
**Pull request by Enning94**
-Pasted the codes for review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>Rabia and Festus product page</title>
+    <link rel="stylesheet" href="style.css">
+
+  </head>
+
+<body>
+  
+  <header id="header">
```
Comment (Enning94) [inline index.html:12]: The header tag is a semantic tag in HTML and it's not appropriate to be used as an ID name.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    float: right;   
+    list-style: none;
+    text-decoration: none;
+    background-color: rgb(254, 159, 43);
+}  
+#form{
+    padding: 20px;
+} 
+    
+
```
Comment (Enning94) [inline style.css:23]: Kindly check and delete all empty spaces/lines in your styling

**Review by Enning94 — COMMENTED** [review]
Check the comments under the review.

**Review by zhadier — APPROVED** [review]
## Hi @Enning94  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**